src/Caching/SqlServer/src/DatabaseOperations.cs

```diff
@@ -77,23 +79,36 @@ public async Task DeleteCacheItemAsync(string key, CancellationToken token = def
         return GetCacheItem(key, includeValue: true);
     }
 
-    public async Task<byte[]?> GetCacheItemAsync(string key, CancellationToken token = default(CancellationToken))
+    public bool TryGetCacheItem(string key, IBufferWriter<byte> destination)
+    {
+        return GetCacheItem(key, includeValue: true, destination: destination) is not null;
+    }
+
+    public Task<byte[]?> GetCacheItemAsync(string key, CancellationToken token = default(CancellationToken))
+    {
+        token.ThrowIfCancellationRequested();
```

Comment:
Isn't this the first thing the overload's going to do anyway?  Seems unnecessary if we're just supplying a default parameter value.

Reply:
this isn't changed code; this was L82 in the original